scripts/generate_docs.py

```diff
@@ -257,9 +257,13 @@ def generate_html(dstpath):
     sourcepath = os.path.join(dstpath, "source")
 
     print("Generating HTML...")
-    cmdline = "sphinx-build -M html %s ../docs"%sourcepath
-    print(cmdline)
-    os.system(cmdline)
+    proc = subprocess.Popen(["sphinx-build", "-M", "html", sourcepath, "../docs" ], stderr=subprocess.PIPE)
```

Comment:
hmm, this might be simpler:
```
result = subprocess.run(f"sphinx-build -M html {sourcepath} ../docs", stderr=subprocess.PIPE)
if result.returncode != 0:
   ...
```

Reply:
Done. 👍